test/ApplePlatformsIntegrationTestApp/ApplePlatformsIntegrationTestApp/main.m

```diff
@@ -0,0 +1,8 @@
+#import <Cocoa/Cocoa.h>
+
+int main(int argc, const char * argv[]) {
+  @autoreleasepool {
+      // Setup code that might create autoreleased objects goes here.
+  }
+  return NSApplicationMain(argc, argv);
```

Comment:
should this be inside @autoreleasepool?  or else, should we just remove 4-6?

Reply:
We can just remove that boilerplate 👍